src/main/java/com/exactpro/th2/codec/csv/CsvCodec.java

```diff
@@ -50,6 +51,9 @@
 public class CsvCodec implements MessageListener<RawMessageBatch> {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(CsvCodec.class);
+    private static final String HEADER_MSG_TYPE = "CsvHeader";
+    private static final String CSV_MESSAGE_TYPE = "Csv_Message";
+    private static final String HEADER_FIELD_NAME = "Header";
```

Comment:
We probably should expose these constants so we can use them in tests

Reply:
In my opinion, exposing such constants to the tests makes changing them more thoughtless. You change a string value and everything still works.
If you use string literals in tests instead you see how many places you affect by changing them